bin/regionEditor.ts

```diff
@@ -0,0 +1,51 @@
+import os from "os";
+import fs from "fs";
+import { PolusBuffer } from "../lib/util/polusBuffer";
+import { RegionList } from "../lib/util/regionList";
+import dns from "dns";
+import util from "util";
+const lookup = util.promisify(dns.lookup);
+import path from "path";
+
+let regionFile: string;
+
+if (process.argv[2] == "set-region" && !!process.argv[5]) {
+  regionFile = process.argv[5];
+} else if (process.argv[2] == "add-server" && !!process.argv[6]) {
+  regionFile = process.argv[6];
+} else {
+  let home = os.homedir();
+  let file = path.join(
+    home,
+    "/AppData/LocalLow/Innersloth/Among Us/regioninfo.dat"
+  );
+  if (fs.existsSync(file)) {
+    regionFile = file;
+  } else {
+    throw new Error("regioninfo.dat not found!");
+  }
+}
+
+function writeBuffer(regionList: PolusBuffer) {
+  fs.writeFileSync(regionFile, regionList.buf, "hex");
+}
+
+(async () => {
+  if (process.argv[2] == "set-region") {
+    let list = new RegionList(
+      process.argv[3],
+      (await lookup(process.argv[4])).address
```

Comment:
Up to you, but might be worth de-duplicating the `await lookup` here in each `if` branch (and maybe checking for an error on lookup, else people might mistype IPs and it'll screw their files up)